americangut/question/ag_question.py

```diff
@@ -0,0 +1,117 @@
+import inspect
+
+import numpy as np
+
+
+class AgQuestion:
+    """A base object class for handling American Gut Data dictionary entries"""
+    true_values = {'yes', 'y', 'true', 1, 1.0, 'Y', 'YES', 'Yes', True}
```

Comment:
I can trim, but there are multiple paths that might end up being cast to yes/no (Like, if you have a Yes/No/Not sure and then you drop the `Not sure` and want to treat it as bool.